src/CommunityToolkit.Maui.Markup/Style.cs

```diff
@@ -86,6 +86,34 @@ public Style<T> Add(params (BindableProperty Property, object Value)[] setters)
 		return this;
 	}
 
+	/// <summary>
+	/// Adds the supplied <paramref name="light"/> and <paramref name="dark"/> values in an <see cref="AppThemeBinding"/>.
+	/// </summary>
+	/// <param name="property">"The <see cref="BindableProperty"/> to style </param>
+	/// <param name="light">"The light value for the <see cref="BindableProperty"/> </param>
+	/// <param name="dark">"The dark value for the <see cref="BindableProperty"/> </param>
+	/// <returns>Style with added setters</returns>
+	public Style<T> Add(BindableProperty property, object light, object dark)
```

Comment:
What do you think about changing the name of this method to something like `AddAppThemeBinding`?

My concern is that the method's API Definition is pretty vague, `Add(BindableProperty, object, object)`, and a dev may not read the variable names (`object light` and `object dark`) and subsequently miss use it.